tools/pom.xml

```diff
@@ -237,6 +236,10 @@
 					<artifactId>maven-jar-plugin</artifactId>
 					<version>3.0.2</version>
 				</plugin>
+				<plugin>
+					<artifactId>maven-war-plugin</artifactId>
+					<version>3.2.2</version>
```

Comment:
Here it's version 3.2.2, while in cldr-apps/pom.xml the version is removed above. Why the difference? In general if version is omitted would we get the most recent?

Reply:
cldr-apps is a child and inherits from the parent.  this way we only have one place to maintain all the version numbers